### ISSUE TITLE
Learning circles

### ISSUE THREAD

**Issue by sofer** [foundersandcoders/coursebook#404]
The [current page on learning circles](https://learn.foundersandcoders.com/course/handbook/learning-circles/) needs attention. I'll make a pr before the next curriculum circle. It also needs a related crib sheet and glossary. I'll make a start on those, too. I'll add notes on why I'm proposing the changes in this issue.

**Comment by sofer** [comment]
Some relevant academic references:
- [Relative effects of three questioning strategies in ill-structured, small group problem solving](https://link.springer.com/article/10.1007%2Fs11251-013-9278-1);
- [Increasing explanatory behaviour, problem-solving, and reasoning within classes using cooperative group work](https://link.springer.com/article/10.1007/s11251-010-9130-9);
- [Scaffolding students’ problem-solving processes in an ill-structured task using question prompts and peer interactions](https://link.springer.com/article/10.1007/BF02504515);
- [Scaffolding the development of problem-solving skills in chemistry](https://pubs.rsc.org/ko/content/articlehtml/2017/rp/c7rp00009j);
- [With a Little Help from My Friends: Scaffolding Techniques in Problem Solving](https://files.eric.ed.gov/fulltext/EJ1052936.pdf);
- [Instructional Methods, Strategies and Technologies to Meet the Needs of All Learners](https://granite.pressbooks.pub/teachingdiverselearners/chapter/scaffolding-2/);
- [Scaffolding Strategy In Mathematics Learning](https://core.ac.uk/download/pdf/11059731.pdf).

TL;DR: Scaffolding (in this case, in the form of initial facilitation and a set of guide questions) can significantly improve problem-solving outcomes. In other words, peer-led groups are going to work better if they are kicked off with a facilitator and a crib sheet of questions.

**Comment by sofer** [comment]
New copy:

---

# Learning circles

## Set up
- Groups of 4. Typically, lasting 90 minutes, with four 20-minute rounds;
- Start with a check-in and end with a check-out;
- Inexperienced groups should have a facilitator present.

## Roles

- Proposer: presents a problem;
- Questioner: asks questions of the proposer;
- Responder: proposes solutions to the proposer;
- Observer: reflects on the problem and the responses of the group.

## Rounds

- The proposer shares their screen and presents the problem that they want to discuss;
- The questioner then proceeds solely by asking questions of the proposer with a view to the whole group gaining a better undertanding of the problem;
- If after 5 minutes or so and solely through the proposer's responses to questioning, the group has not yet gained an adequate understanding of the problem, it is the responder's role to try to navigate the proposer to a solution to the problem by more direct means; 
- After about 15 minutes, it is the observer's role to reflect on the problem and on the behaviour of the group;
- If there is a facilitator present, they can add their own reflections on whether the problem was at a suitable level, whether the group as a whole was supportive of each other, whether each participant stayed in role, and on the solution. They should also make sure the group is on schedule. If there is no facilitator present, this role can be taken by the observer;
- When the round comes to an end, each group member takes on a new role and the process is repeated until each member of the group has played all four roles.

Aims:
1. To give each participant an opportunity to get the support of the group with a problem that they are working on;
2. To give each participant practice in developing questioning strategies to aid problem solving;
3. To give each participant practice at communicating proposed solutions;
4. To give each participant practice at thinking reflectively about their learning processes;
5. To ensure that each participant gets an equal opportunity to develop their problem-solving skills;
6. To develop the capacity of all participants to run a strucutured study group without a facilitator present.

## Tools

- Question crib sheet, that the questioner can use as a scaffold for their questioning;
- Glossary of terms.

**Comment by sofer** [comment]
I have been using the current "learning circles" copy as a starting point for both coaching groups and mock interviews, for which in both cases it has proved inadequate. The role-based mock interview groups have worked particularly well. They do not quite follow this new template, but they have switched my thinking back to the value of roles. So, roles are back in this new guidance and we have added an explicit role of "questioner", which continues to be something I suspect the value of but cannot get early-stage learners to get the hang of. The plan now is to add a list of "questions you can ask, even if you yourself do not understand what is going on". the hope is that these questions will be the "scaffolding" that a large research literature seems to indicate is critical in helping learners cultivate adequate problem-solving strategies in a wide variety of classroom contexts.

**Comment by sofer** [comment]
Latest commit simplifies things again.
